docker/docker-compose.ci.yml

```diff
@@ -43,52 +39,77 @@ services:
     image: unlock
     user: node
     working_dir: /home/unlock/unlock-app
+    ports:
+      - 3000:3000
```

Comment:
Note that we're now exposing the ports so it is easier to debug locally with docker.